rules/lint-fix-format.ham

```diff
@@ -27,14 +28,21 @@ rule lffFiles_sh {
   DEPENDS $(<) : $(<)_fix $(<)_lint ;
 }
 
+SHFMT_PARAMS ?= -i 2 -ci -bn -ln=bash -bn=false ;
+SHELLCHECK_PARAMS ?= --shell=bash --format=gcc ;
+SHELLCHECK_PARAMS += -e SC1091 ; # 'Not following: ... was not specified as input'
+# This is better than disabling SC1091, but oh so slowww :(
+# SHELLCHECK_PARAMS += -x --source-path="$(HAM_HOME)" --source-path="$HAM_HOME/bin" ;
+
 actions together lffLintFiles_sh {
-  . "$HAM_HOME/bin/ham-bash-lib.sh"
-  . "$HAM_HOME/bin/ham-lint-fix-lib.sh"
-  ham_lint_fix_main --lint --check-format sh "$(>)"
+  HAM_NO_VER_CHECK=1 . hat shell_linter > /dev/null
+  set -x
```

Comment:
we need set -ex otherwise shfmt failing will be ignored

Reply:
Done in commit e83508c

Originally I had omitted the `-e` on purpose since it seemed like running `ham lint` would still have ham produce a build error when a file had lint issues, and also had the advantage of continuing to run the linter on all the provided files so you could see all of the lint errors from multiple files in one run.

Adding the -e flag will now fail on the first error so you will only see your lint issues one file at a time.